packages/devextreme/js/ui/load_panel.d.ts

```diff
@@ -125,6 +125,7 @@ export interface dxLoadPanelOptions extends dxOverlayOptions<dxLoadPanel> {
      * @docid
      * @default 90
      * @default 60 &for(Material)
+     * @default auto &for(Fluent)
```

Comment:
Perhaps, we should write auto in single quotes.

Reply:
@alexanderPolosatov

Reply:
>Perhaps, we should write auto in single quotes.

Yes. Strings should be wrapped to quotes. @EugeniyKiyashko @iBat